New fix for signature breaks unit tests since it now tests for length

Either change the `assertFalse` or modify the `b'wrong'` to a 64 character length word.

How can it be landed with the unit test failure?
```
============================================== FAILURES ===============================================
____________________________________ Test_bitcoin.test_msg_signing ____________________________________

self = <lib.tests.test_bitcoin.Test_bitcoin testMethod=test_msg_signing>

    def test_msg_signing(self):
        msg1 = b'Chancellor on brink of second bailout for banks'
        msg2 = b'Electrum'
    
        def sign_message_with_wif_privkey(wif_privkey, msg):
            txin_type, privkey, compressed = deserialize_privkey(wif_privkey)
            key = regenerate_key(privkey)
            return key.sign_message(msg, compressed)
    
        sig1 = sign_message_with_wif_privkey(
            'L1TnU2zbNaAqMoVh65Cyvmcjzbrj41Gs9iTLcWbpJCMynXuap6UN', msg1)
        addr1 = '15hETetDmcXm1mM4sEf7U2KXC9hDHFMSzz'
        sig2 = sign_message_with_wif_privkey(
            '5Hxn5C4SQuiV6e62A1MtZmbSeQyrLFhu5uYks62pU5VBUygK2KD', msg2)
        addr2 = '1GPHVTY8UD9my6jyP4tb2TYJwUbDetyNC6'
    
        sig1_b64 = base64.b64encode(sig1)
        sig2_b64 = base64.b64encode(sig2)
    
        self.assertEqual(sig1_b64, b'H/9jMOnj4MFbH3d7t4yCQ9i7DgZU/VZ278w3+ySv2F4yIsdqjsc5ng3kmN8OZAThgyfCZOQxZCWza9V5XzlVY0Y=')
        self.assertEqual(sig2_b64, b'G84dmJ8TKIDKMT9qBRhpX2sNmR0y5t+POcYnFFJCs66lJmAs3T8A6Sbpx7KA6yTQ9djQMabwQXRrDomOkIKGn18=')
    
        self.assertTrue(verify_message(addr1, sig1, msg1))
        self.assertTrue(verify_message(addr2, sig2, msg2))
    
>       self.assertFalse(verify_message(addr1, b'wrong', msg1))

lib/tests/test_bitcoin.py:82: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
lib/bitcoin.py:508: in verify_message
    public_key, compressed = pubkey_from_signature(sig, h)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

sig = b'wrong'
h = b'\x80OA\x0c\x11@:\xa5\xbe\xca\x83\xb3\x00\x18\xe73\xe2\xd0Ea\xdd\xe1\xdb\xd4V\xbb\x13G\xeb\x08Z\xc5'

    def pubkey_from_signature(sig, h):
        if len(sig) != 65:
>           raise Exception("Wrong encoding")
E           Exception: Wrong encoding

lib/bitcoin.py:597: Exception
```

--- a/lib/tests/test_bitcoin.py
+++ b/lib/tests/test_bitcoin.py
@@ -79,7 +79,7 @@ class Test_bitcoin(unittest.TestCase):
         self.assertTrue(verify_message(addr1, sig1, msg1))
         self.assertTrue(verify_message(addr2, sig2, msg2))
 
-        self.assertFalse(verify_message(addr1, b'wrong', msg1))
+        self.assertRaises(Exception, verify_message, addr1, b'wrong', msg1)
         self.assertFalse(verify_message(addr1, sig2, msg1))
 
     def test_aes_homomorphic(self):

